pom.xml

```diff
@@ -4672,7 +4672,23 @@
    </reporting>
 
    <profiles>
-
+      <profile>
+         <id>unsupported-test</id>
+         <build>
+            <plugins>
+               <plugin>
+                  <groupId>org.apache.maven.plugins</groupId>
+                  <artifactId>maven-surefire-plugin</artifactId>
+                  <configuration>
+                     <excludes>
+                        <!-- example: **/RocksDBReaderTest.java,**/LevelDBStoreTest.java,**/RocksDBStoreTest.java -->
+                        <exclude>${unsupported-test-excludes}</exclude>
```

Comment:
I don't think you need a separate profile @diegolovison, just use it normally and define the property to be empty by default (and preferably name it `infinispan.test.something`). 

Actually I don't think you need to change anything in the POM, couldn't list the exclusions in a file and run Maven with [`-Dsurefire.excludesFile=yourfile`](http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#excludesFile)?